tests_config/run_spec.js

```diff
@@ -95,7 +95,8 @@ function stripLocation(ast) {
         key === "raw" ||
         key === "comments" ||
         key === "parent" ||
-        key === "prev"
+        key === "prev" ||
+        key === "__location"
```

Comment:
What do you think about replacing this with `["loc", "range", "raw", "comments", "parent", "prev", "__location"].indexOf(key) !== -1`?